examples/records/records.bal

```diff
@@ -1,27 +1,46 @@
 import ballerina/io;
 
-// Defines a `record` type named `Coord`.
-type Coord record {
-    int x;
-    string y;
+// Defines a `closed record` type named `Student`. It only allows fields that are specified.
+type Student record {|
+    string name;
+    int age;
+|};
+
+// Defines an `open record` type named `Person`. It allows fields other than those specified.
+type Person record {
+    string name;
+    int age;
 };
 
 public function main() {
-    // Creates a `record`, specifying values for its fields.
-    record { int x; string y; } r = {
-        x: 1,
-        y: "2"
+    // Creates an `open record` by specifying values for its fields.
+    record {string name; int age;} stu = {
```

Comment:
Shall we use another variable name instead of `stu`?

Reply:
Used `person`, `employee` and `student` instead.